packages/google_maps_flutter/google_maps_flutter_web/lib/src/markers.dart

```diff
@@ -5,26 +5,26 @@
 part of '../google_maps_flutter_web.dart';
 
 /// This class manages a set of [MarkerController]s associated to a [GoogleMapController].
-class MarkersController extends GeometryController {
+class MarkersController<T, O> extends GeometryController {
```

Comment:
I suspect we need a `class AdvancedMarkersController extends MarkersController` instead of generics here too.